README.md

```diff
@@ -24,29 +24,48 @@ You may install the package using npm install command:
 npm install ibm_db
 ```
 
-### Important Environment Variables
+> For more installation details please refer:  [INSTALLAION GUIDE](https://github.com/ibmdb/node-ibm_db/blob/master/INSTALL.md)
+
+
+### Important Environment Variables and Download Essentials 
 
 `IBM_DB_HOME :`
-```
-USE: Set this environment variable if you want to avoid downloading of odbc/clidriver from the IBM provided URL
-     or from the internet.
 
-How : Set `IBM_DB_HOME` environment variable to a pre-installed `db2 client or server installation directory`.
-```
+- USE: Set this environment variable if you want to avoid downloading of clidriver from the [IBM Hosted URL](#downloadCli) or from the internet.
```

Comment:
Yes in the latest update I changed it into single line paragraph. Is it still not showing?